spacy/tests/pipeline/test_spancat.py

```diff
@@ -400,15 +407,15 @@ def mixed_zero_suggester(docs, *, ops=None):
     fix_random_seed(0)
     nlp = English()
     spancat = nlp.add_pipe(
-        "spancat",
+        name,
```

Comment:
Just noting that the conflict emerged here. Instead of using `test_zero_suggester`, it's now using the `test_mixed_zero_suggester`.

Reply:
Looks like the test suite is failing now though?

Reply:
It is waiting on the next release of thinc.